### ISSUE TITLE
Hey awesome Addon! Taminus!

### ISSUE THREAD

**Issue by eXCepTIoNGER** [xternal7/ultrawidify#4]
I love your addon ultrawidify, on youtube it works perfectly but is it possilbe to implement it into the pornhub player? ;)

thanks for your answer in advance :)

**Comment by xternal7** [comment]
**TL;DR: Maybe** (if Pornhub uses HTML5 player - if it uses flash, then no dice), **but I make no guarantees.**

At the moment, this is on the very end of the list of sites and features that I consider a priority.

I'm currently doing a major rewrite of the extension so adding new sites would be easier (at the moment, it's a bad mess). While doing so, I'm also going to include some settings page for people to add their own rules that would do so. (And a quick how-to page explaining how to do that). 
This would, among other things, allow people to easily make the extension work with sites that I either can't or won't support, or sites that I don't consider a top priority.
